test/SILOptimizer/moveonly_nonescaping_closures.swift

```diff
@@ -46,18 +46,21 @@ func a(x: __shared M) {
     borrow(x)
 }
 
-func b(x: __owned M) { // expected-error{{}}
-    clodger({ borrow(x) }, consume: x) // expected-note{{}} expected-note{{}}
+func b(x: __owned M) {
+    clodger({ borrow(x) }, consume: x)
+    // expected-error @-1 {{'x' was consumed but it is illegal to consume a noncopyable immutable capture of an escaping closure. One can only read from it}}
 }
 
 func c(x: __owned M) {
     clodger({ borrow(x) })
     borrow(x)
     consume(x)
+    // expected-error @-1 {{'x' was consumed but it is illegal to consume a noncopyable immutable capture of an escaping closure. One can only read from it}}
```

Comment:
This is not an escaping closure, so this code should be valid without diagnostics.